### PR TITLE
Design polish for top of navbar

### LINKED ISSUE #777: Update UI for top-level navigation bar
**What needs to be done? Why does it need to be done?**
This ticket involves the redesign of the current ND top-level navigation. This task involves only new UI and visual updates of the existing ND views - no changes to the actual page hierarchy.

Mocks: https://www.figma.com/file/H08AQJSaAIRxfeMP4iCUPh/UP-MVP?node-id=1521%3A43

Existing nav:

New nav: 

On hover:

Dropdown (the options here should be "Facilities", "Community", and "Programming" to match what currently exists in ND instead of what is shown here):

The default page order should be: 
1. Facilities
2. Community 
3. Programming

When the dropdown is showing, the current page should be at the top of the list, followed by the other two pages in the default order. For example, if the user is currently on the "Community" page, the dropdown should be in this order: Community, Facilities, Programming. 

A new navigation bar enables:
- Extensibility to match UP's future structure
- Hierarchies the page more based on the user
- The navigation to be less obtrusive and take up less room
- Focus on accessibility

@soniachokshi to add more details

**Additional context**
CC @Catacola this could be a good FE starter task since it's UI polish building atop our current ND dash, unless it conflicts with how you are thinking through the overall infrastructure for UP.

### PR DESCRIPTION
**Pull request by Catacola**
## Description of the change

Update all of the little design changes needed to bring the top bar of the core view across the line for #777. Figma files: https://www.figma.com/file/H08AQJSaAIRxfeMP4iCUPh/UP-MVP?node-id=1521%3A43

Hopefully when you load it up, it looks like this:
Screen Shot 2021-03-11 at 4 15 37 PM

Known issues:
- ~Responsive design is not yet implemented~ DONE!
- The arrow on the dropdown on the right is clipped. However, this seems to be an issue with the library itself: https://reactstrap.github.io/components/dropdowns/

## Type of change

- [x] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

Closes #777.

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/core/CoreNavigation.tsx [hunk 1]
```diff
@@ -21,34 +21,42 @@ import CoreSectionSelector from "./CoreSectionSelector";
...
+  // @ts-ignore
+  const pageOptions = navigationLayout[currentSection] ?? [];
+
   return (
-    <div className="CoreNavigation">
-      <ul className="nav-left">
+    <nav className="CoreNavigation">
+      <div className="CoreNavigation__left">
+        <div className="CoreNavigation__logo">
+          <img
```
Comment (daschi) [inline src/core/CoreNavigation.tsx:43]: Currently I think this logo links to the `/community/goals` page, do we want to keep that behavior in this polish?

File: src/core/PageTemplate.scss [hunk 2]
```diff
@@ -4,4 +4,14 @@
   background: $core-background;
   padding-left: 13.5rem;
   padding-right: 13.5rem;
+
```
Comment (daschi) [inline src/core/PageTemplate.scss:7]: I think we may want to move the background style on line 4 to  apply to the parent class `.page-container`. On pages like `facilities/goals` the content is not always tall enough to reach the footer and is resulting in some extra whitespace at the bottom

**Review by daschi — APPROVED** [review]
How exciting 🎆  it's looking really great! I had a few small comments but LGTM!!

File: src/core/PageTemplate.scss [hunk 3]
```diff
@@ -4,4 +4,14 @@
...
+  @media screen and (max-width: 1440px) {
+    padding-left: 4rem;
+    padding-right: 4rem;
+  }
+
+  @media screen and (max-width: 1366px) {
```
[the remark this answers is not in the document]
Comment (Catacola) [reply src/core/PageTemplate.scss:13]: Good idea.

File: src/core/PageTemplate.scss (hunk 2 above)
Comment (Catacola) [reply src/core/PageTemplate.scss:7]: Good catch!

File: src/core/CoreNavigation.tsx (hunk 1 above)
Comment (Catacola) [reply src/core/CoreNavigation.tsx:43]: Done!

File: src/core/PageTemplate.scss (hunk 2 above)
Comment (Catacola) [reply src/core/PageTemplate.scss:7]: Update: fixed by removing `min-height: 100vh` from `.page-container`

**Pull request merged**